app/src/main/java/ch/epfl/sdp/Player.java

```diff
@@ -131,9 +132,6 @@ public void run() {
                     t.schedule(shrinkAoeRadius, (long) ((Shrinker) i).getShrinkTime() * 1000);
                     break;
                 case "Scan":
-                    //((Scan) i).showPlayersLocation();
-                    break;
-                default:
```

Comment:
Here why we remove the Scan case and default.

Reply:
You're right, this is my bad. I removed it to test something and I forgot to add it back. Thank you very much